# 개별 장애물 데이터
geometry_msgs/Point obstacle_global
float32 ob_height
float32 ob_width
float32 a_ob_distance